recipes/r-vioplot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = '0.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-vioplot
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: vioplot_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/vioplot_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/vioplot/vioplot_{{ version }}.tar.gz
+  sha256: 601f043630999e0498f0fba213d3d3fefd6ff0abf8fdb22119199b3d8d58939b
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-sm
+
+  run:
+    - r-base
+    - r-sm
+
+test:
+  commands:
+    - $R -e "library('vioplot')"           # [not win]
+    - "\"%R%\" -e \"library('vioplot')\""  # [win]
+
+about:
+  home: http://wsopuppenkiste.wiso.uni-goettingen.de/~dadler
+  license: BSD
+  summary: 'A violin plot is a combination of a box plot and a kernel density plot.'
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+    - rasmusagren
```

Comment:
In addition to yourself, could you please add the [conda-forge R maintainers](https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml)? This makes it easier for us to update the recipe in the future.

```
    - johanneskoester
    - bgruening
    - daler
    - jdblischak
```